gdcdictionary/schemas/subject.yaml

```diff
@@ -151,15 +152,22 @@ properties:
     description: >
       First three digits of the resident's zipcode.
     type: string
-    
+
  # Prop originated from SSR project
   version:
     description: >
       The data version as determined by the data submitter.
-    type: string   
+    type: string
 
 
   studies:
     $ref: "_definitions.yaml#/to_many"
   projects:
     $ref: "_definitions.yaml#/to_one_project"
+
+# Prop originated for Longcovid and Monkeypox data
+
+  unique_de-id_identifier:
+    description: >
+        "Unique and continuous identifier for subject"
```

Comment:
it's not clear to me from the description what this ID is

Reply:
The unique_de-id_identifier is a unique identifier for any subject/patient. This is unique per subject and it is continous all through system.

Reply:
can we just make it the `submitter_id` then? 🙂 that's the conventional unique ID across Gen3

Reply:
Actually, this id is provided by the healthcare organization and we would be utilizing it for research. I would recommend separating the submitter id from this so it doesn't impact other projects

Reply:
Actually, this id is provided by the healthcare organization and we would be utilizing it for research. I would recommend separating the submitter id from this submitter id as we are currently using submitter_id to get the name of the org and get the submission date of that particular subject as shown below.

<img width="617" alt="Screen Shot 2023-01-25 at 1 52 37 AM" src="https://user-images.githubusercontent.com/70549768/214400774-9afc5057-f894-4379-8f8f-6574740b27c2.png">

Reply:
it's your call so i will approve the PR to unblock you, but this sounds to me like the org and date should be stored in their own properties if you think they are useful information, and the unique ID should be stored in the existing property that is meant for IDs that are unique across the project => submitter_id

Reply:
That's a good insight. Let me talk to team internally if we can take this approach in the next iteration. Can you approve this PR whenever you can?

Reply:
sure
i already approved it, the tests didn't pass though